docs/core/setup/install-sdk.md

```diff
@@ -0,0 +1,144 @@
+
+::: zone-end
+
+## Install with Visual Studio Code
+
+Visual Studio Code is a powerful and lightweight source code editor that runs on your desktop. VS Code is available for Windows, macOS, and Linux.
+
+While VS Code doesn't come with .NET Core support, adding .NET Core support is simple.
+
+01. [Download and install Visual Studio Code](https://code.visualstudio.com/Download).
+01. [Download and install the .NET Core SDK](https://dotnet.microsoft.com/download/dotnet-core/3.0).
+01. [Install the C# extension from the VS Code marketplace](https://marketplace.visualstudio.com/items?itemName=ms-vscode.csharp).
+
+::: zone pivot="os-windows"
+
+## Install with PowerShell automation
+
+The [dotnet-install scripts](../tools/dotnet-install-script.md) are used for automation and non-admin installs of the SDK. You can download the script from the [dotnet-install script reference page](../tools/dotnet-install-script.md).
+
+The script defaults to installing the latest [long term support (LTS)](https://dotnet.microsoft.com/platform/support/policy/dotnet-core) version, which is currently .NET Core 2.1. To install the current release of .NET Core (3.0), run the script with the following switch:
+
+```powershell
+dotnet-install.ps1 -Channel 3.0
+```
+
+::: zone-end
+
+::: zone pivot="os-linux,os-macos"
+
+## Install with bash automation
+
+The [dotnet-install scripts](../tools/dotnet-install-script.md) are used for automation and non-admin installs of the SDK. You can download the script from the [dotnet-install script reference page](../tools/dotnet-install-script.md).
+
+The script defaults to installing the latest [long term support (LTS)](https://dotnet.microsoft.com/platform/support/policy/dotnet-core) version, which is currently .NET Core 2.1. To install the current release of .NET Core (3.0), run the script with the following switch:
+
+```bash
+./dotnet-install.sh -c Current
+```
+
+::: zone-end
+
+## Docker
+
+Containers provide a lightweight way to isolate your application from the rest of the host system, sharing just the kernel, and using resources given to your application.
+
+.NET Core can run in a Docker container. Official .NET Core Docker images are published to the Microsoft Container Registry (MCR) and are discoverable at the [Microsoft .NET Core Docker Hub repository](https://hub.docker.com/_/microsoft-dotnet-core/). Each repository contains images for different combinations of the .NET (SDK or Runtime) and OS that you can use.
+
+Microsoft provides images that are tailored for specific scenarios. For example, the [ASP.NET Core repository](https://hub.docker.com/_/microsoft-dotnet-core-aspnet/) provides images that are built for running ASP.NET Core apps in production.
+
+For more information about using .NET Core in a Docker container, see [Introduction to .NET and Docker](../docker/introduction.md) and [Samples](https://github.com/dotnet/dotnet-docker/blob/master/samples/README.md).
+
+## Next steps
```

Comment:
On MVC land (https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-mvc-tutorial?branch=master), next steps is a single link. Should we use See also here?

```suggestion
## See also
```

Reply:
I do prefer **See also**, but I dont know if we're supposed to use it? All the guidance I've seen has deemed **Next steps** as the header title. That MVC suggestion you pointed to though is *only* for the tutorial type. Other pages can have more. For example the overview type says 1-3 next steps.